docs/gameplay-sessions/gameplay-sessions-reference.mdx

```diff
@@ -0,0 +1,347 @@
+      "setPassCalls": number, // An integer representing the number of set pass calls
+    }
+  },
+  "state": GameObjectState[], // An array of objects holding the state of game objects (see following section)
+  "inputs": {
+    "keyboard": KeyboardInput[], // An array of objects holding keyboard input (see following section)
+    "mouse": MouseInput[] // An array of objects holding mouse input (see following section)
+  }
+```
+
+#### GameStateObject Schema
+
+Within the `state` field, an array of objects represents the state of game objects on this tick. Depending on the
+[options you have configured](/gameplay-sessions/gameplay-sessions-usage#advanced-options) for recording, some objects and state in your scene may not be present in this
+list. Each object has the following schema:
+
+```json
+{
+  "id": number, // An integer representing the unique identifier of this object
+  "path": string, // A string representing the path of this object in the scene hierarchy
+  "scene": string, // A string representing the scene this object is in
+  "tag": string, // A string representing the tag of this object
+  "layer": string, // A string representing the layer of this object
+  "rendererCount": number, // An integer representing the number of renderers on this object
+  "screenSpaceBounds": {
+    "center": {
+      "x": number, // A float representing the x coordinate of the center of the bounds
+      "y": number, // A float representing the y coordinate of the center of the bounds
+      "z": number // A float representing the z coordinate of the center of the bounds
+    },
+    "extents": {
+      "x": number, // A float representing the x coordinate of the extents of the bounds
+      "y": number, // A float representing the y coordinate of the extents of the bounds
+      "z": number // A float representing the z coordinate of the extents of the bounds
+    }
+  },
+  "worldSpaceBounds": {
+    "center": {
+      "x": number, // A float representing the x coordinate of the center of the bounds
+      "y": number, // A float representing the y coordinate of the center of the bounds
+      "z": number // A float representing the z coordinate of the center of the bounds
+    },
+    "extents": {
+      "x": number, // A float representing the x coordinate of the extents of the bounds
+      "y": number, // A float representing the y coordinate of the extents of the bounds
+      "z": number // A float representing the z coordinate of the extents of the bounds
+    }
+  },
+  "position": {
+    "x": number, // A float representing the x coordinate of the position of the object
+    "y": number, // A float representing the y coordinate of the position of the object
+    "z": number // A float representing the z coordinate of the position of the object
+  },
+  "rotation": {
+    "x": number, // A float representing the x component of the rotation of the object
+    "y": number, // A float representing the y component of the rotation of the
```

Comment:
Noticed there may be some issues in these comments. Maybe I should just remove a lot of these comments... some of these fields are pretty clear

Reply:
I think removing these comments is a good idea 👍🏻 
This part of the docs is going to change constantly I think

Reply:
Removed!